From 21486201778c284e2609c72acbab0df3d3a8d270 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Tue, 30 Apr 2019 07:20:15 +0900
Subject: [PATCH 1/2] spp_nfv: add master lcore in status msg

To know which one is master lcore is required for considering core
assignment if master is shared among mulit processes. This update is to
add master lcore ID in response of status command.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/nfv/nfv_status.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/nfv/nfv_status.c b/src/nfv/nfv_status.c
index d29e083..7e3dbd4 100644
--- a/src/nfv/nfv_status.c
+++ b/src/nfv/nfv_status.c
@@ -55,6 +55,8 @@ append_lcore_info_json(char *str,
 		uint8_t lcore_id_used[RTE_MAX_LCORE])
 {
 	int i;
+	sprintf(str + strlen(str), "\"master-lcore\":%d,",
+			rte_get_master_lcore());
 	sprintf(str + strlen(str), "\"lcores\":[");
 	for (i = 0; i < RTE_MAX_LCORE; i++) {
 		if (lcore_id_used[i] == 1)
-- 
2.17.1

